packages/ilios-common/addon/components/learningmaterial-manager.js

```diff
@@ -137,9 +137,7 @@ export default class LearningMaterialManagerComponent extends Component {
     this.startDate = learningMaterial.startDate;
     this.endDate = learningMaterial.endDate;
 
-    const meshDescriptors = await learningMaterial.get('meshDescriptors');
-    this.terms = meshDescriptors.slice();
-
+    this.terms = await learningMaterial.get('meshDescriptors');
```

Comment:
I should have noticed this the first time, sorry, this should be
```suggestion
    this.terms = await learningMaterial.meshDescriptors;
```
and remove the old school Ember pattern of using `get` here.